samples/outlook/40-attachments/attachments-compose.yaml

```diff
@@ -79,6 +79,17 @@ script:
             }
             return url;
         }
+
+        function getAttachmentTypeMessage(attachmentType) {
+            switch(attachmentType) {
+                case Office.MailboxEnums.AttachmentType.Cloud:
+                    return "Attachment is stored in a cloud location.";
+                case Office.MailboxEnums.AttachmentType.File:
+                    return "Attachment is a file.";
+                case Office.MailboxEnums.AttachmentType.Item:
+                    return "Attachment is an Exchange item.";
+            }
```

Comment:
If this is pulled out to a separate function, does the sample still make sense when ported to the reference docs? Conversely, does this function as a standalone sample give enough context to show what types provide the enum?

Reply:
Thanks for raising this. I've been going back and forth on this one, but after giving it more thought, I've integrated this into the `get` function for better sample comprehension.